src/scrape_up/flipkart/flipkart_file.py

```diff
@@ -321,3 +321,130 @@ def computer(self):
 
         except Exception as e:
             return None
+
+
+"""
+        Create an instance of `FlipkartTees` class.
+
+    ```python
+    cloth = FlipkartTees()
+    ```
+
+    | Methods               | Details                                                            |
+    | --------------------- | ------------------------------------------------------------------ |
+    | `.scrape()`           | Returns the list of t-shirts with other relevant info              |
+    | `.range()`            | Returns the list of t-shirts between a particular price range.     |
+    | `.minrating()`        | Returns the list of t-shirts havinga minimum given rating.         |
+    | `.gendermale()`       | Returns the list of t-shirts which are for males.                  |
+    | `.genderfemale()`     | Returns the list of t-shirts which are there for females.          |
+    | `.size()`             | Returns the list of tshirts havning a particular size.             |
+
+    """
+
+
+
+class FlipkartTees:
```

Comment:
Add this class in a separate file named `flipkart_clothing.py` and add this method to that class. This is the convention we follow, one class in a file.